style.css

```diff
@@ -4,14 +4,96 @@
 
 body {
   margin: 0;
-  font-size: 12rem;
 }
 
 ul {
-  padding-left: 0
+  padding-left: 0;
   list-style: none;
 }
 
 img {
   max-width: 100%;
+  border-radius: 50%;
+}
+
+.banner {
+  background-color: #4fc0ff;
+  padding: 0 1rem;
+  border-radius: 2rem 2rem 0 0;
```

Comment:
Sorry for the confusion. The styling in Newline makes all pages look like they have a rounded border, but the navbar should not have any rounded edges.